### ISSUE TITLE
[Feature Request] Add a helper class for visually hidden content

### ISSUE THREAD

**Issue by blakewatson** [vuetifyjs/vuetify#10914]
### Problem to solve
This is for content which should be visually hidden, but remain accessible to assistive technologies such as screen readers. This can be useful in situations where additional visual information or cues (such as meaning denoted through the use of color) need to also be conveyed to non-visual users. Another example is if you have an icon button but you would like to provide screen-reader-only text to describe the button.

### Proposed solution
I say just grab the `.sr-only` and `.sr-only-focusable` utilities from Bootstrap 4. https://getbootstrap.com/docs/4.4/utilities/screen-readers/

**Comment by johnleider** [comment]
Resolved in https://github.com/vuetifyjs/vuetify/pull/11258

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).
